core/modules/czar/python/proxy.py

```diff
@@ -49,10 +62,19 @@ class Lock:
     acquiring a lock on the table. mysql-proxy can service other
     requests when it is waiting on mysql queries that it knowingly
     initiates in its C-layer, whereas it treats its Lua plugin code as a
-    single-threaded black-box."""
-    createTmpl = "CREATE TABLE IF NOT EXISTS %s (chunkId INT, code SMALLINT, message CHAR(255), timeStamp FLOAT) ENGINE=MEMORY;"
+    single-threaded black-box.
+    """
+
+
+    createTmpl = "CREATE TABLE IF NOT EXISTS %s " \
+                 "(chunkId INT, code SMALLINT, message CHAR(255), " \
+                 "severity ENUM ('INFO', 'ERROR'), timeStamp FLOAT) ENGINE=MEMORY;"
```

Comment:
fyi, ";" at the end of queries sent through c api is not needed, it is only needed in mysql client. I'd remove it for cleanness.